action.yml

```
@@ -17,78 +17,75 @@ inputs:
     description: "API account token to retrieve your remote Codacy configuration for the project being analyzed"
   codacy-api-base-url:
     required: false
-    description: "Codacy's API URL, to retrieve your remote Codacy configuration"
+    description: "Codacy API URL to retrieve your remote Codacy configuration"
   format:
     required: false
-    description: "Format of the output file"
+    description: "Output file format"
   output:
     required: false
-    description: "Send analysis results to an output file"
+    description: "Path to a file to save the analysis results"
   directory:
     required: false
-    description: "The directory to be analyzed"
+    description: "Directory to analyze"
   parallel:
     required: false
     description: "Number of tools to run in parallel"
   max-tool-memory:
     required: false
-    description: "Maximum of allowed memory for each tool execution"
+    description: "Maximum allowed memory for each tool (bytes)"
```

Small clarification:

```suggestion
    description: "Maximum allowed memory for running each tool (bytes)"
```